packages/graalvm/src/test/kotlin/elide/runtime/gvm/js/node/AbstractJsModuleTest.kt

```diff
@@ -39,20 +39,25 @@ internal abstract class AbstractJsModuleTest<T: GuestIntrinsic> : AbstractJsIntr
     // nothing
   }
 
-  protected val polyglotEngine by lazy {
-    PolyglotEngine {
-      install(JavaScript)
-      environment { configureEnvironment() }
-      vfs { configureVfs() }
-    }.acquire()
+  @OptIn(DelicateElideApi::class)
+  protected val polyglotContext by lazy {
+    engine.acquire()
   }
```

Comment:
The base class (`AbstractDualTest`) already provides an API to configure the engine used in tests, no need to have this duplicate init logic here (it was breaking some test behavior because it didn't line up with the base class). Simply obtain a context instance for tests from the base engine, and apply extra configuration in the dedicated callback.